### ISSUE TITLE
Fatal bug: ReferenceError: _sync is not defined

### ISSUE THREAD

**Issue by create3000** [lmaccherone/node-localstorage#87]
Version `"node-localstorage": "^3.0.4",` throws the following error:

```
App threw an error during load
ReferenceError: _sync is not defined
    at LocalStorage._init (/Users/holger/Desktop/X_ITE/sunrize/node_modules/node-localstorage/LocalStorage.js:166:11)
    at new LocalStorage (/Users/holger/Desktop/X_ITE/sunrize/node_modules/node-localstorage/LocalStorage.js:114:14)
    at Object.<anonymous> (/Users/holger/Desktop/X_ITE/sunrize/src/sunrize/Application/Application.js:16:22)
```

Shouldn't it be `this._sync()`?

**Comment by remisture** [comment]
Same problem here. Anyone working on a fix?

**Comment by lmaccherone** [comment]
I will fix it tonight. Sorry
________________________________

**Comment by lmaccherone** [comment]
Should be fixed in 3.0.5.
